### PR TITLE
Edits to HIP-869

### PR DESCRIPTION
**Pull request by Mark-Swirlds**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by jsync-swirlds** [comment]
Would it be helpful to include the more thorough specification language for the proto files in this document?
For reference, the PR to update the proto files is [here](https://github.com/hashgraph/hedera-protobufs/pull/346) with a separate branch to show the [rendered documentation](https://github.com/hashgraph/hedera-protobufs/tree/temp-DAB-Update-Spec-Backup/documents/api/services) generated from those updates.

File: HIP/hip-869 [hunk 1]
```diff
@@ -0,0 +1,374 @@
+---
```
Comment (netopyr) [inline HIP/hip-869:1]: I am confused. Was this file added by accident?

File: HIP/hip-869.md [hunk 2]
```diff
@@ -58,63 +58,48 @@ Adopting a two-phase strategy, this approach facilitated the earlier release of
...
+**Council** -  Hedera Council 
 
-**Node Operator** - Technical administrator of Hedera Consensus Nodes
+**Node Operator** - Administrator of Hedera Consensus Nodes
 
-**Add Node to Address Book**
 
-***User Story:*** As a council member, I want the ability to submit signed HAPI transactions to add a new consensus node to the Hedera network upon the next maintenance window, so that I can perform operations independently.
+***User Stories:*** 
+1. The Council submits signed HAPI transactions to add a new consensus node to the Hedera network upon the next maintenance window, for automated management of Hedera's Address book.
```
Comment (netopyr) [inline HIP/hip-869.md:67]: Strictly speaking, this does not qualify as a user story IMO. A user story should answer who wants to do what and for what reason. It should focus on the problem, not the solution. The standard structure helps (but is not required).

File: HIP/hip-869.md [hunk 3]
```diff
@@ -58,63 +58,48 @@ Adopting a two-phase strategy, this approach facilitated the earlier release of
...
 
-***Acceptance:*** When a valid council member initiates a HAPI transaction to add a new node, then the network should acknowledge the transaction and perform an update to the network’s Address Book at the next maintenance window.
+*Acceptance: When the council initiates a HAPI transaction to add a new node, then the network acknowledges the transaction and performs the update to the network’s Address Book at the next maintenance window.*
 
-**Remove Node from Address Book**
+2. The Council submits signed HAPI transactions to remove a new consensus node to the Hedera network upon the next maintenance window, for automated management of Hedera's Address book.
```
Comment (netopyr) [inline HIP/hip-869.md:71]:
```suggestion
2. The Council submits signed HAPI transactions to remove a consensus node from the Hedera network upon the next maintenance window, for automated management of Hedera's Address book.
```

File: HIP/hip-869.md [hunk 4]
```diff
@@ -58,63 +58,48 @@ Adopting a two-phase strategy, this approach facilitated the earlier release of
...
 
-**Update Node’s Public Key**
+8. A Node Operator submits a signed HAPI transaction that modifies a node’s X509 certificate hash within the Address Book.
 
-***User Story:*** As a council member, I value the ability to submit a signed HAPI transaction that modifies a node’s public key within the Address Book used for signing.
+*Acceptance: When a Node Operator submits a HAPI transaction to modify a node's associated X509 certificate hash, then the network acknowledges the transaction and performs the update the network’s Address Book at the next maintenance window.*
 
-***Acceptance:*** When a council member submits a HAPI transaction to modify a node's associated Public Key, then the network should acknowledge the transaction and perform update the network’s Address Book at the next maintenance window.
+9.  A node operator uses the Network Management Tool (NMT) to automatically adjust firewall configurations based on transactions associated with address book changes, so that I can perform operations independently.
```
Comment (netopyr) [inline HIP/hip-869.md:99]: Does this belong to the HIP?

File: HIP/hip-869.md (hunk 2 above)
Comment (Mark-Swirlds) [reply HIP/hip-869.md:67]: I'll change back.

**Pull request closed without merging**